shared/k8s/debug.go

```diff
@@ -50,6 +50,8 @@ func FetchValidatorPrivateKeys(clientset *kubernetes.Clientset) (map[string]stri
 	return validatorKeysMap, nil
 }
 
+// ADDPR: add the following functions in a separate PR: FetchServicerPrivateKeys and FetchAppPrivateKeys
```

Comment:
Can you update the `Makefile` explaining what `ADDPR` is? Or is this a `TODO`?

Reply:
This is a TODO that is not going to be committed to the code base: it just indicates that a separate PR will be opened to cover the required feature addressed in this POC PR. The term is selected to be distinct from anything in the Makefile, making it easier to ensure all the PRs required to implement the POC have been opened.

Reply:
In the `Makefile` we have a couple `*_IN_THIS_COMMIT` type comments.

![Screenshot 2023-07-05 at 3 52 06 PM](https://github.com/pokt-network/pocket/assets/1892194/83315a09-7528-45a4-8b80-6f0dc496b31e)

Wdyt do you think of renaming this to `ADD_IN_THIS_PR` so:
1.  it's more self-explanatory
2. We explain it in the Makefile
3. We can one day have a CI that blocks merging this to main